src/main/java/com/amazon/ion/_Private_Trampoline.java

```diff
@@ -0,0 +1,17 @@
+package com.amazon.ion;
+
+import java.io.IOException;
+
+/**
+ * <b>NOT FOR APPLICATION USE. This class may be removed at any time.</b>
+ */
+public final class _Private_Trampoline {
```

Comment:
FYI, you could fairly trivially rewrite this in Kotlin and mark it as `internal`. You don't even need to include the class.

E.g. `_Private_Trampoline.kt`
```kotlin
package com.amazon.ion

/**
 * **NOT FOR APPLICATION USE. This method may be removed at any time.**
 * Trampoline to the non-public `Timestamp.toString(Int)` method.
 */
internal fun print(timestamp: Timestamp, maximumDigits: Int): String {
    return timestamp.toString(maximumDigits)
}
```

Reply:
Done, thanks